src/h3/api/numpy_int/_binding.py

```diff
@@ -21,13 +21,23 @@
 
 import numpy as np
 
+try:
+    from numpy.typing import NDArray
+    ArrayType = NDArray[np.uint64]
+except ImportError:
+    ArrayType = np.ndarray
```

Comment:
The numpy.typing module is new in v1.20: https://numpy.org/devdocs/reference/typing.html